doc/source/authentication.rst

```diff
@@ -191,3 +191,64 @@ You can specify this list of usernames in your `config.yaml`:
        users:
          - user1
          - user2
+
+Advanced: adding your own Ingress object
```

Comment:
IMO, Ingress is unrelated to authentication / security and should remain in advanced.

Reply:
yep, already fixing it, didn't think you'd notice that so quickly haha